plugins/services/src/js/containers/volume-detail/VolumeDetail.js

```diff
@@ -54,6 +54,10 @@ class VolumeDetail extends React.Component {
       />
     );
 
+    const sizeLabel = volume.type === 'External'
```

Comment:
Airbnb/javascript gives a warn here
![screen shot 2017-03-28 at 14 36 05](https://cloud.githubusercontent.com/assets/186223/24428641/f1409e92-13c3-11e7-9d3d-a63784190a84.png)

We've agreed on ternaries and we have eslint rules in place